### DISCUSSION TITLE
Why should I use this instead of directly from DoD Cyber Exchange?

### DISCUSSION THREAD

**Discussion by TheDanHealy** [ComplianceAsCode/content#11859] [General]
I'm not understanding some of the "why" and hope the community can help. The DoD Cyber Exchange produces the official STIGs for RHEL 8. Why should I use this project's repo (https://github.com/RedHatOfficial/ansible-role-rhel8-stig) instead of downloading the official STIG for Ansible (https://public.cyber.mil/stigs/downloads/)?

**Comment by Mab879** [comment]
The primary reason I see (as a maintainer of this project) is the this project normally has much better coverage than DISA does on their automation. Saving you time on remediation or creating your own playbooks.

I would be interested in hearing the opinion of others.
